### PR TITLE
run the 2to3 script

### PR DESCRIPTION
**Pull request by Gustry**
```
etienne@:~/dev/cpp/QGIS (master) $ cd /Users/etienne/dev/cpp/QGIS
etienne@:~/dev/cpp/QGIS (master) $ scripts/2to3 -x absolute_import -wn /Users/etienne/dev/inasafe/inasafe/safe_extras/parameters/
```
Search and replace `from qgis.PyQt` to `from PyQt5` because we want to stay generic outside of QGIS

Not ready for merge

### CODE REVIEW

**Comment by Gustry** [comment]
@ismailsunni Should we merge this to a new branch `master-py3-qt5`?
Or maybe just tag the current master branch with py2-qt4 and then create a new legacy master branch if we need to fix something after on py2 version (so we keep the master with the current develop version).

What do you think about this PR? Maybe I need some reading about str and unicode in Py 3, it's confusing.

**Comment by Gustry** [comment]
Note I haven't tried it yet. Just trusting travis. I will test later when I will work on InaSAFE 5. Just preparing this repo to work soon on that task.

File: parameters/boolean_parameter.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# coding=utf-8
```
Comment (ismailsunni) [inline parameters/boolean_parameter.py]: No we don't need the this comment anymore?

File: parameters/dict_parameter.py [hunk 2]
```diff
@@ -36,7 +35,7 @@ def __getitem__(self, key):
 
     def keys(self):
         """Helper method to make DictParameter behave like dict"""
-        return self._value.keys()
+        return list(self._value.keys())
```
Comment (ismailsunni) [inline parameters/dict_parameter.py]: New syntax? I thought it's already a list.

**Comment by ismailsunni** [comment]
> Should we merge this to a new branch master-py3-qt5?
> Or maybe just tag the current master branch with py2-qt4 and then create a new legacy master branch if we need to fix something after on py2 version (so we keep the master with the current develop version).

I prefer the second option

**Comment by ismailsunni** [comment]
> Note, I haven't tried this PR yet. Just trusting travis and tests you wrote a few years ago. I will test later when I will work on InaSAFE 5. Just preparing this repo to work soon on that task.

Beside the unit test, you can also run the example, and see if it's looking good. see: https://github.com/inasafe/parameters/blob/master/parameters/qt_widgets/test/main.py

File: parameters/boolean_parameter.py (hunk 1 above)
Comment (Gustry) [reply parameters/boolean_parameter.py]: It's UTF8 by default in Python 3.
https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python3

File: parameters/dict_parameter.py (hunk 2 above)
Comment (Gustry) [reply parameters/dict_parameter.py]: Not anymore in python 3:

```
etienne@:~/Downloads $ python3.6
Python 3.6.3 (default, Oct  5 2017, 10:19:42) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.37)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> my_dict = {1:'a', 2:'b'}
>>> my_dict.keys()
dict_keys([1, 2])
>>> list(my_dict.keys())
[1, 2]
>>> 
```

The `dict_keys` object is still iterable. So it's ok in a for loop:
```
for key in my_dict.keys():
```

Comment (ismailsunni) [reply parameters/dict_parameter.py]: Ah I see... it's now a `<class 'dict_keys'>` in python 3.6

File: parameters/boolean_parameter.py (hunk 1 above)
Comment (ismailsunni) [reply parameters/boolean_parameter.py]: Ah I see...
I should follow your porting process so that I can also get new knowledge of python 3 :)

Thanks

Comment (Gustry) [reply parameters/boolean_parameter.py]: It's the big change in Py3, UTF8 by default and unicode too. No more encoding problem ;-)

**Comment by Gustry** [comment]
Closing this one in favor of https://github.com/inasafe/parameters/pull/34

**Pull request closed without merging**